docs/content/en/docs-dev/contribution-guidelines/development.md

```diff
@@ -8,34 +8,30 @@ description: >
 
 ## Prerequisites
 
-Only `bazel` is required to build and test `PipeCD` project, but you don't need to install `bazel` directly.
-Instead of that we are using [`bazelisk`](https://github.com/bazelbuild/bazelisk) for automatically picking a good version of Bazel for building `PipeCD` project.
-
-You can install `bazelisk` by `go get` as the following:
-```
-go get github.com/bazelbuild/bazelisk
-```
-
-or directly install its [binary](https://github.com/bazelbuild/bazelisk/releases) from the release page.
-For more information, you might want to read the [installation notes of `bazelisk`](https://github.com/bazelbuild/bazelisk#requirements).
+- [Go 1.17](https://go.dev/)
+- [Docker Decktop](https://www.docker.com/products/docker-desktop/)
 
 ## Repositories
 - [pipecd](https://github.com/pipe-cd/pipecd): contains all source code and documentation of PipeCD project.
-- [manifests](https://github.com/pipe-cd/manifests): contains all automatically generated release manifests for both `Piped` and `Control Plane` components.
 - [examples](https://github.com/pipe-cd/examples): contains various generated examples to demonstrate how to use PipeCD.
 
-## Build and test with Bazel
+## Commands
+
+- `make build/backend`: builds all binaries of all backend modules.
+- `make build/frontend`: builds the static files for frontend.
+
+- `make test/backend`: runs all unit tests of backend.
+- `make test/frontend`: runs all unit tests of backend.
+- `make test/integration`: runs integration tests.
+
+- `make run/pipecd`: runs Control Plane locally.
+- `make run/piped`: runs Piped Agent locally.
+- `make run/frontend`: runs frontend locally.
```

Comment:
should be make run/console to unify with docs, another option is make run/web since we have make web-dev currently 🤔

Reply:
```markdown
- `make run/web`: runs web console locally.
```